client/src/components/EmailDialog.vue

```diff
@@ -112,11 +112,16 @@ export default {
       this.subject = experiment;
       this.body = `Experiment: ${this.currentSession.experiment}
 Scan: ${this.currentSession.name}`;
-      if (this.note) {
+      if (this.notes) {
         this.body = `${this.body}
-Note:
-${this.note}
+Notes:`;
+        for (let i = 0; i < this.notes.length; i++) {
+          let note = this.notes[i];
+          this.body = `${this.body}
+${note.creator.first_name} ${note.creator.last_name}: ${note.created}
+${note.note}
```

Comment:
```javascript
this.body += this.notes.map((note) => `${note.creator.first_name} ${note.creator.last_name}: ${note.created}\n${note.note}`).join('\n');
```

I think that's more idiomatic and readable. We can let the linter figure out the line breaks hopefully.